src/test/java/com/debatetimer/domain/MemberTest.java

```diff
@@ -0,0 +1,30 @@
+package com.debatetimer.domain;
+
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class MemberTest {
+
+    @Nested
+    class Validate {
+
+        @ParameterizedTest
+        @ValueSource(ints = {0, Member.NICKNAME_MAX_LENGTH + 1})
+        void 닉네임은_정해진_길이_이내여야_한다(int length) {
+            assertThatThrownBy(() -> new Member("f".repeat(length)))
+                    .isInstanceOf(IllegalArgumentException.class)
+                    .hasMessage(String.format("닉네임은 1자 이상 %d자 이하여야 합니다", Member.NICKNAME_MAX_LENGTH));
```

Comment:
[제안] 자바 13부터 추가된 `formatted()`를 사용해보시는 건 어떨까요?

```suggestion
                    .hasMessage("닉네임은 1자 이상 %d자 이하여야 합니다".formatted(Member.NICKNAME_MAX_LENGTH), Member.NICKNAME_MAX_LENGTH));
```